lib/warmups.rb

```diff
@@ -0,0 +1,32 @@
+module Warmups
+
+def self.stock_picker(stocks)
+  result = {buy: nil, sell: nil, profit: nil}
```

Comment:
raise an error if the argument is not an `Array` and if the length of stocks array is less than 2